cfgov/ask_cfpb/wagtail_hooks.py

```diff
@@ -97,3 +98,44 @@ def register_export_menu_item():
 @hooks.register('register_admin_urls')
 def register_export_url():
     return [url('export-ask', export_data, name='export-ask')]
+
+
+@hooks.register('after_create_page')
+def create_answer_id(request, page):
+    """
+    Create an incremented Answer ID for a new AnswerPage and attach it.
+
+    Also create a sister-language page to keep languages in sync.
+    """
+    if isinstance(page, AnswerPage) and page.answer_base is None:
+        new_answer_base = Answer(
+            last_user=request.user,
+            question=page.title)
```

Comment:
Do we still need the Answer to store the question?

Reply:
No, but I thought it would be useful to devs who may have reason to traverse Answer. We should make Answer's __str__ method return question, so there's some context to an Answer object beyond it's primary key.